### PR TITLE
Add router and "Info" page

### PR DESCRIPTION
**Pull request by vlrsmlkv**
Info should contain a manual for pu-2 processor and any other useful information

### CODE REVIEW

File: src/components/FileField.jsx [hunk 1]
```diff
@@ -36,13 +36,13 @@ const FileField = ({onChange}) => {
       ? `Выбрано файлов: ${filesNumber}`
       : (filesNumber === 1) 
         ? `${filesNameList}`
-        : <>Выбрать файлы <Paperclip/></>
+        : <>Choose file <Paperclip/></>
```
Comment (OUghTMS) [inline src/components/FileField.jsx:39]: file(s)

**Review by OUghTMS — CHANGES_REQUESTED** [review]

File: src/App.jsx [hunk 2]
```diff
@@ -1,54 +1,30 @@
...
-}
+import React from "react";
+import Main from "./pages/Main";
+import Info from "./pages/Info";
+import NavBar from "./components/NavBar";
+
+import { BrowserRouter as Router, Route, Redirect, Switch } from "react-router-dom";
+
+const App = () => (
+  <div className="root-page">
```
[the remark this answers is not in the document]
Comment (vlrsmlkv) [reply src/App.jsx:9]: fixed

File: src/App.jsx [hunk 3]
```diff
@@ -1,54 +1,30 @@
...
+        <div className="App">
+          <Switch>
+            <Route exact path="/">
+              <Main />
+            </Route>
+            <Route exact path="/info">
+              <Info />
+            </Route>
+            <Route>
+              <Redirect to="/"/>
```
[the remark this answers is not in the document]
Comment (vlrsmlkv) [reply src/App.jsx:21]: everything works fine
but you need to check it with me again 👉👈 🥺

File: src/components/NavBar.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import { Disc, Info } from 'react-feather';
+
+const NavBar = () => {
+  return (
+    <div className="nav-bar">
+      <NavLink 
+        className="nav-bar-button"
+        exact to="/"
```
[the remark this answers is not in the document]
Comment (vlrsmlkv) [reply src/components/NavBar.jsx:10]: added

File: src/components/NavBar.jsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+          fontWeight: "bold",
+          borderColor: "black"
+        }}
+      >
+        <Disc className="nav-bar-button-icon"/> Main
+      </NavLink>
+
+      <NavLink 
+        className="nav-bar-button"
+        exact to="/info"
```
[the remark this answers is not in the document]
Comment (vlrsmlkv) [reply src/components/NavBar.jsx:22]: added

File: src/components/NavBar.jsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+      <NavLink 
+        className="nav-bar-button"
+        exact to="/info" 
+        activeClassName="selected" 
+        activeStyle={{
+          fontWeight: "bold",
+          borderColor: "black"
+        }}
+      >
+        <Info className="nav-bar-button-icon"/>Info
```
[the remark this answers is not in the document]
Comment (vlrsmlkv) [reply src/components/NavBar.jsx:29]: added

File: src/components/FileField.jsx (hunk 1 above)
Comment (vlrsmlkv) [reply src/components/FileField.jsx:39]: done

File: src/pages/Info.jsx [hunk 7]
```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+import { File, Info as Infoicon, Upload, BookOpen} from 'react-feather';
+
+const Info = () => {
+
+  return (
+    <div className="info-page-container">
+      
+      <div className="info-page-block">
```
[the remark this answers is not in the document]
Comment (vlrsmlkv) [reply src/pages/Info.jsx]: done

File: src/pages/Info.jsx [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+                <ul className="unordered">
+                  <li className="additional">In case you want each processed file to be displayed on a separate excel sheet, tick the "Show each file as a separate sheet" checkbox.</li>
+                </ul>
+              </li>
+              <li><b>Step 2.</b> Click on "Show preview" button if needed.</li>
+              <li><b>Step 3.</b> Download .xlsx file with a list of employees and information about them.</li>
+            </ul>
+          </div>
+      </div>
+      <div className="info-page-block">
```
[the remark this answers is not in the document]
Comment (vlrsmlkv) [reply src/pages/Info.jsx]: done

File: src/pages/Info.jsx [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+        infoText={"The pu-2 application is a tool developed to help in processing data prepared by the Client in the form of pu-2 forms."}
+      />
+
+      <InfoBlock
+        icon={<File className="info-page-icon"/>}
+        header={"Pu-2 form"}
+        infoText={"PU-2 form refers to personalized accounting documents and contains information about the hiring and dismissal of employees."}
+      />
+
+      <InfoBlock
```
Comment (OUghTMS) [inline src/pages/Info.jsx:24]: Let's make InfoBlock a HoC where infoText is a children component

File: src/pages/Info.jsx [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+              <li><b>Step 2.</b> Click on "Show preview" button if needed.</li>
+              <li><b>Step 3.</b> Download .xlsx file with a list of employees and information about them.</li>
+            </ul>
+        }
+      />
+
+      <InfoBlock
+        icon={<Upload className="info-page-icon"/>}
+        header={"Upload file requirements"}
+        infoText={
```
Comment (OUghTMS) [inline src/pages/Info.jsx:43]: Here same. infoText should be children

**Review by OUghTMS — CHANGES_REQUESTED** [review]

File: src/pages/Info.jsx (hunk 10 above)
Comment (vlrsmlkv) [reply src/pages/Info.jsx]: done

File: src/pages/Info.jsx (hunk 9 above)
Comment (vlrsmlkv) [reply src/pages/Info.jsx:24]: done

**Comment by vlrsmlkv** [comment]
Fix router !!! @OUghTMS

**Pull request merged**